python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -275,7 +275,7 @@ def runAssociation(self, sensorRef):
             solarSystemObjectTable=None,
             diffIm=sensorRef.get(diffType + "Diff_differenceExp"),
             exposure=sensorRef.get("calexp"),
-            warpedExposure=sensorRef.get(diffType + "Diff_warpedExp"),
+            template=sensorRef.get(diffType + "Diff_warpedExp"),
```

Comment:
Should this be `_templateExp`? Not that I think we've run `ApPipeTask` in a while... 😛

Reply:
There are also references to `doWriteWarpedExp` in this file.